app/src/androidTest/java/org/mozilla/fenix/ui/HomeScreenTest.kt

```diff
@@ -153,4 +173,46 @@ class HomeScreenTest {
             verifyWelcomeHeader()
         }
     }
+
+    @Test
+    fun verifyPocketFunctionalityTest() {
+        featureSettingsHelper.setRecentTabsFeatureEnabled(false)
+        featureSettingsHelper.setRecentlyVisitedFeatureEnabled(false)
+
+        homeScreen {
+        }.dismissOnboarding()
+
+        homeScreen {
+            verifyThoughtProvokingStories(true)
+            verifyStoriesByTopic(true)
+        }.openThreeDotMenu {
+        }.openCustomizeHome {
+            clickPocketButton()
+        }.goBack {
+            verifyThoughtProvokingStories(false)
+            verifyStoriesByTopic(false)
+        }
+    }
+
+    @Test
+    fun verifyCustomizeHomepageButtonTest() {
```

Comment:
Similar, but I'd just shorten to  `verifyCustomizeHomepageTest()`

Reply:
Done !